src/main/java/core/basesyntax/service/suppliers/FigureSupplier.java

```diff
@@ -0,0 +1,48 @@
+package core.basesyntax.service.suppliers;
+
+import core.basesyntax.Circle;
+import core.basesyntax.Figure;
+import core.basesyntax.IsoscelesTrapezoid;
+import core.basesyntax.Rectangle;
+import core.basesyntax.RightTriangle;
+import core.basesyntax.Square;
+import java.util.Random;
+
+public class FigureSupplier {
+    public Figure getRandomFigure() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        int figureIndex = new Random().nextInt(4);
+        switch (figureIndex) {
+            case 0:
+                return new Circle(colorSupplier.getRandomColor(), getRandomInt());
```

Comment:
```suggestion
                return gerRandomCircle();
```
lets also make private methods to get each figure, this way your code is easier to read